app/graphql/mutations/save_account_mutation.rb

```diff
@@ -0,0 +1,12 @@
+# frozen_string_literal: true
+
+module Mutations
+  # GraphQL mutation to update the current user account
+  class SaveAccountMutation
+    def call(user, args, _ctx)
```

Comment:
It seems easier to me to replicate Devise's update functionality than working around all the things the controller needlessly does (need to create a request and response object, handle the render call, ...).

[update](https://github.com/plataformatec/devise/blob/master/app/controllers/devise/registrations_controller.rb#L44-L63) doesn't look like that much, when stripping away the stuff we won't need (flash, respond_with, bypass_signin, ...).